test/spec/features/modeling/CreateElementsSpec.js

```diff
@@ -212,6 +215,28 @@ describe('features/modeling - create elements', function() {
       expect(elements[0]).to.equal(shape);
     }));
 
+
+    it('should not have fractional coordinates', inject(function(elementFactory, modeling) {
+
+      // given
+      var shape = elementFactory.createShape({
+        id: 'shape',
+        x: 110,
+        y: 110,
+        width: 105,
+        height: 105
+      });
+
+      var elements = modeling.createElements(shape, position, parentShape);
+
+      // then
+      expect(pick(elements[0], ['x', 'y'])).to.eql({

```

Comment:
You can use the `to.have.position` matcher here.

Reply:
🤯

Reply:
Cool stuff! :)